pkg/data/management/machinedriver_data.go

```diff
@@ -61,17 +61,6 @@ var driverDefaults = map[string]map[string]string{
 	Vmwaredriver:    {"vcenterPort": "443"},
 }
 
-type machineDriverCompare struct {
-	builtin            bool
-	addCloudCredential bool
-	url                string
-	uiURL              string
-	checksum           string
-	name               string
-	whitelist          []string
-	annotations        map[string]string
-}
-
 func addMachineDrivers(management *config.ManagementContext) error {
```

Comment:
Now that I think about it, couldn't we just register a feature handler here?

Reply:
Not here, but the machine driver controller could watch for feature flags and turn the corresponding driver on or off.
The thing is: AFAIK the Harvester node driver is the only one for which a feature flag exists.